src/tests/ftest/osa/offline_drain.py

```diff
@@ -29,7 +29,10 @@ def setUp(self):
         self.ior_test_sequence = self.params.get(
             "ior_test_sequence", '/run/ior/iorflags/*')
         # Recreate the client hostfile without slots defined
-        self.hostfile_clients = write_host_file(self.hostlist_clients, self.workdir)
+        self.hostfile_clients = write_host_file(self.hostlist_clients, self.workdir, None)
```

Comment:
No need to pass `slots=None` anymore
```suggestion
        self.hostfile_clients = write_host_file(self.hostlist_clients, self.workdir)
```

This is probably a bad (minor) merge